src/routes/governance/hooks.ts

```diff
@@ -78,17 +81,36 @@ export const useVoteInformation = ({
     () => (requiredMajority ? Number(requiredMajority) * 100 : 100),
     [requiredMajority]
   );
-  const yesTokens = React.useMemo(
-    () => Number(proposal.votes.yes.totalTokens),
-    [proposal.votes.yes.totalTokens]
-  );
-  const noTokens = React.useMemo(
-    () => Number(proposal.votes.no.totalTokens),
-    [proposal.votes.no.totalTokens]
-  );
+
+  const noTokens = React.useMemo(() => {
+    if (!proposal.votes.no.votes) {
+      return 0;
+    }
+    const totalNoVotes = proposal.votes.no.votes.reduce(
+      (prevValue: number, newValue: Proposal_proposal_votes_no_votes) => {
+        return prevValue + Number(newValue.party.stake.currentStakeAvailable);
+      },
+      0
+    );
+    return Number(addDecimal(new BigNumber(totalNoVotes), 18));
+  }, [proposal.votes.no.votes]);
+
+  const yesTokens = React.useMemo(() => {
+    if (!proposal.votes.yes.votes) {
+      return 0;
+    }
+    const totalYesVotes = proposal.votes.yes.votes.reduce(
+      (prevValue: number, newValue: Proposal_proposal_votes_yes_votes) => {
+        return prevValue + Number(newValue.party.stake.currentStakeAvailable);
```

Comment:
Use bignumber

Reply:
Hm using bignumber here has quite a cascading affect of requiring lots of other changes. Lets do as a follow up

Reply:
https://github.com/vegaprotocol/token-frontend/issues/971